dts/arm/st/u3/stm32u3.dtsi

```diff
@@ -233,6 +235,43 @@
 			interrupts = <66 0>;
 			status = "disabled";
 		};
+
+		adc1: adc@42028000 {
+			compatible = "st,stm32-adc";
```

Comment:
This should be changed to "st,stm32n6-adc" because like N6, U3 ADC has no clock setting (inside the ADC itself).